lib/SPIRV/OCLTypeToSPIRV.cpp

```diff
@@ -1,4 +1,4 @@
-﻿//===- OCLTypeToSPIRV.cpp - Adapt types from OCL for SPIRV ------*- C++ -*-===//
+//===- OCLTypeToSPIRV.cpp - Adapt types from OCL for SPIRV ------*- C++ -*-===//
```

Comment:
It was in UTF-8 encoding. Now it is in ASCII. First 3 bytes (byte order mark) have been removed.